include/wx/utils.h

```diff
@@ -156,6 +156,9 @@ WXDLLIMPEXP_BASE bool wxIsPlatform64Bit();
 // Get machine CPU architecture
 WXDLLIMPEXP_BASE wxString wxGetCpuArchitectureName();
 
+// Get native machine CPU architecture
+WXDLLIMPEXP_BASE wxString wxGetNativeCpuArchitectureName();
```

Comment:
I wonder if should keep adding global functions for all the possible fields of `wxPlatformInfo`. I'd be happy enough to just have the new field there, but it does mean that we'd have to somehow initialize it in platform-specific way...

Reply:
I think it would make sense to only add to `wxPlatformInfo` but the platform specific implementation would have to be put somewhere and the `utils.cpp` is usually a good fit. Maybe it would make sense to just not make the global function part of the interface and just use it internally in `wxPlatformInfo` ?